examples/examples_nodejs_test.go

```diff
@@ -760,3 +760,34 @@ func TestAccAuthenticationModeMigration(t *testing.T) {
 
 	integration.ProgramTest(t, &test)
 }
+
+func TestAccManagedNodeGroupCustom(t *testing.T) {
```

Comment:
Can you add this to our CI configs as well?

Reply:
It's included in this one: https://github.com/pulumi/pulumi-eks/actions/runs/9501673502/job/26188058924?pr=1196#step:23:1419

Reply:
Ah yes, the name matching is not a exact match:
https://github.com/pulumi/pulumi-eks/blob/92465dce68dd560ae48fd0211cae9a74b4ee19e4/Makefile#L196C3-L197C178